examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p><em>Suggested change:</em></p><p> </p><p>Read more about <em>warmup </em>technique in <a href="https://arxiv.org/abs/1812.01187v2" target="_blank">this paper</a>.</p><p> </p><p>~~~</p><p> </p><p>Also consider merging this cell with the one with <code>domain</code> description.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>